packages/neo-one-website/docs/2-advanced-guides/10-configuration-options

```diff
@@ -0,0 +1,41 @@
+---
+slug: config-options
+title: Configuration Options
+---
+NEO•ONE compiler configuration options
+
+Configure NEO•ONE for your project.
+
+## Config File
+
+An [initialized environment](https://neo-one.io/tutorial#Setup-for-the-Tutorial) will have a ` .onerc ` file within the project root.
+
+### Example NEO•ONE configuration file
+
+The `.onerc` configuration file could look something like this:
+```json
+{
+  "codegen": {
+    "language": "typescript",
+    "framework": "react",
+    "browser":  true
+  },
+  "paths": {
+    "contracts": "one/contracts",
+    "generated": "one/generated"
+  },
+}
+```
+
+### `codegen`
+
+Keys with `codegen` allow you to configure which `language`, front-end `framework` and whether to compile with `browser` compatibility.
+* `language`: `string` =  `typescript` | `javascript`
+* `framework`: `string` =  `none` | `react` | `vue` | `angular`
+* `browser`:  `boolean` = `true` | `false`
```

Comment:
While the language and framework may be self explanatory, the browser key needs to be explained in more detail.  We don't want people to use this option unless they absolutely need to.  Use what I wrote in my previous comment under the browser key.

Reply:
I also think we should specify the defaults somehow.

Reply:
added defaults:
https://github.com/neo-one-suite/neo-one/pull/1177/files#diff-172ea3c052d9b136cba8549a61c6fd9dR33